tools/gatk_genomicsdbimport_genotypegvcfs.cwl

```diff
@@ -0,0 +1,53 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: gatk_genomicsdbimport_genotypegvcfs
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/gatk:4.0.12.0'
+  - class: ResourceRequirement
+    ramMin: $(inputs.ram * 1000)
+    coresMin: $(inputs.cpu)
+baseCommand: []
+arguments:
+  - position: 0
+    shellQuote: false
+    valueFrom: >-
+      /gatk --java-options "-Xms$((inputs.ram - 5) * 500)m -Xmx$((inputs.ram - 5) * 1000)m"
+      GenomicsDBImport
+      -L $(inputs.interval.path)
+      --interval-padding 5
+      --genomicsdb-workspace-path genomicsdb
+      --batch-size 50
+      --reader-threads 5
+  - position: 10
+    shellQuote: false
+    valueFrom: >-
+      && /gatk --java-options "-Xms$(Math.floor(inputs.ram*500/1.074-1))m -Xmx$(Math.floor(inputs.ram*1000/1.074-1))m"
+      GenotypeGVCFs
+      -R $(inputs.reference_fasta.path)
+      -D $(inputs.dbsnp_vcf.path)
+      -L $(inputs.interval.path)
+      --only-output-calls-starting-in-intervals
+      -V gendb://genomicsdb
```

Comment:
bye bye weird tar step!

Reply:
It was indeed not needed for a tool written like this. GATK has a tar step because their pipeline can use different genotyping tools so they need to pass it between tasks. We're only using the one genotyper and in the same tool so no passing is necessary.